src/config_types.ts

```diff
@@ -84,6 +84,7 @@ export interface Context {
     readonly cluster: string;
     readonly user: string;
     readonly name: string;
+    readonly namespace?: string;
```

Comment:
Should this not be just `namespace` like the others and set to `undefined` if not set?